### DISCUSSION TITLE
Checklist boxes should be made larger

### DISCUSSION THREAD

**Discussion by 2137mail** [standardnotes/forum#3104] [Suggestions]
2 issues.

1) On mobile it is very difficult to hit the checkmark on the list. Most of the time my fingers hit the text for editing.
=> make the squares to hit to check a line bigger 
2) If i have an empty entry (by mistake or because i deleted the text of it) the empty entry remains. So I end up with a list with several empty entries
=> either automatically delete an empty entry or have the option to delet an entry (any)

Thx.

**Comment by Pendyploo** [comment]
I have this problem too when using the checklist note type. If I delete the contents of an entry call mom I can't delete the checkbox and end up with many empty check boxes

**Comment by effieeee** [comment]
Empty checklist items can be deleted by hitting **Enter/Return** on the keyboard 🙏
